src/elexmodel/models/GaussianElectionModel.py

```diff
@@ -282,3 +291,6 @@ def get_aggregate_prediction_intervals(
         )
 
         return PredictionIntervals(aggregate_data.lower.round(decimals=0), aggregate_data.upper.round(decimals=0))
+
+    def get_conformalization_data_agg(self):  # Note that precinct level % conformalization data
```

Comment:
do you mind moving this function up to `get_conformalization_data_unit` also please comment on the next line

Reply:
done